src/main/java/fr/polytech/sim/log/TimestampedLoggerDecorator.java

```diff
@@ -0,0 +1,18 @@
+package fr.polytech.sim.log;
+
+import java.time.Instant;
+import java.util.Date;
+
+public class TimestampedLoggerDecorator implements Logger {
+    private final Logger logger;
+    TimestampedLoggerDecorator(Logger logger) {
+        this.logger = logger;
+    }
+
+
+    @Override
+    public void log(String format, Object... args) {
+        format = Date.from(Instant.now()).toString() + ":" + format;
```

Comment:
toString n'est pas nécessaire, mais tout à fait OK.
```suggestion
        format = Date.from(Instant.now()) + ":" + format;
```